main.sh

```diff
@@ -107,40 +99,40 @@ function xml_dump {
 
 function all_sch { # arg($1:group)
     # is loaded by default so data must be loaded
-    data=($(sed 's/://g' $SCHEDULE))
```

Comment:
originally `2utf8` was developed for batti, when time went on it occured that it can be skip it, Since we need [0-9] data only. Now days `2utf8` is helpful when thing break!